### PR TITLE
Fix setting the default value for Kdump target on UI

### PR DESCRIPTION
**Pull request by imobach**
Fix bug bsc#935756.

### CODE REVIEW

File: src/include/kdump/uifunctions.rb [hunk 1]
```diff
@@ -263,12 +263,16 @@ def SetUpKDUMP_SAVE_TARGET(target)
...
         pos = Builtins.search(parse_target, "/")
         pos1 = -1
-        Ops.set(
-          @KDUMP_SAVE_TARGET,
-          "target",
-          Builtins.substring(parse_target, 0, Ops.subtract(pos, 1))
-        )
-        parse_target = Builtins.substring(parse_target, Ops.add(pos, 2))
+        if pos.zero? # Support for the old '/var/crash' style
+          Ops.set(@KDUMP_SAVE_TARGET, "target", "file")
```
Comment (ancorgs) [inline src/include/kdump/uifunctions.rb]: Why use `Ops` here? Is not `@KDUMP_SAVE_TARGET` a regular hash?

File: test/include/uifunctions.rb [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  end
+
+  subject(:client) { DummyYast::KdumpClient.new }
+
+  describe "#SetUpKDUMP_SAVE_TARGET" do
+    context "when target has no type" do
+      let(:target) { "/var/crash" }
+
+      it "sets KDUMP_SAVE_TARGET as 'file' with path '/var/crash'" do
+        expect(subject.SetUpKDUMP_SAVE_TARGET(target)).to eq(true)
```
Comment (ancorgs) [inline test/include/uifunctions.rb]: it's confusing that you use `subject` sometimes and `client` others

File: src/include/kdump/uifunctions.rb (hunk 1 above)
Comment (imobach) [inline src/include/kdump/uifunctions.rb]: I only wanted to fix the code without changing anything else (or mixin 'styles'). IMHO, this function should be rewritten (but not now).

File: test/include/uifunctions.rb (hunk 2 above)
Comment (imobach) [inline test/include/uifunctions.rb]: You're right. Fixed.

File: Rakefile [hunk 3]
```diff
@@ -3,4 +3,5 @@ require "yast/rake"
 Yast::Tasks.configuration do |conf|
   #lets ignore license check for now
   conf.skip_license_check << /.*/
+  conf.obs_target = "SLE_12"
```
Comment (ancorgs) [inline Rakefile]: ??

**Comment by ancorgs** [comment]
The whole method screams for some refactoring, but LGTM for the time being.

**Comment by imobach** [comment]
I agree. But for the time being I'd like to focus on fixing more bugs.

**Pull request merged**